Rewrite contributors entry page with quick fixes (#381)
cncf/tag-contributor-strategy: website/content/contributors/_index.md

Though, contributing to the CNCF ecosystem is not just about coding. There are many other ways to contribute to the CNCF ecosystem, including writing documentation, creating tutorials, bringing ideas to meetings, organizing meetups, and more.

## Open Source Beginners

Are you new to open source? If so, we recommend checking out our comprehensive guide, ["Start Contributing to Open Source"](getting-started.md). This guide is filled with helpful tips and tricks to help you get started with contributing to open source projects. You'll learn about communities and projects, how to find them, how to conform to community standards, and much more.

## Contribution Opportunities

### Mentorship Programs

The Cloud Native Computing Foundation participates in various mentoring programs, including:

- [LFX Mentorship](https://github.com/cncf/mentoring/tree/master/lfx-mentorship) (previously known as Community Bridge) by the Linux Foundation
- [Google Summer of Code](https://github.com/cncf/mentoring/tree/master/summerofcode) (GSoC)
- [Google Season of Docs](https://github.com/cncf/mentoring/tree/master/seasonofdocs) (GSoD)

> - [Outreachy](https://github.com/cncf/mentoring/tree/master/outreachy)

Reviewer: These links all are broken, did the pages move?

Author: Hmm, right.

I actually just move this content from the bottom of the page, but haven't actually checked the links.

Good catch.

Gonna fix it

Author: Copy&pasted the same content from Mentors WG document now.

- [CNCF Landscape](https://landscape.cncf.io/) - a map of all CNCF projects, along with their maturity level and their category
- [CLOTributor](https://clotributor.dev/) - a tool that helps you find beginner-friendly tasks in CNCF projects
- [Projects section below](#projects) lists all CNCF projects, with their primary language and brief information on contributing to them

### Technical Advisory Groups (TAGs)

CNCF Technical Advisory Groups (TAGs) are responsible for overseeing and coordinating the interests of end users and/or projects within a specific area of focus. These groups hold open meetings that are accessible to anyone without any obligations. You are welcome to participate in discussions or simply observe the group's activities. If you are interested in contributing to a TAG, you can find more information in their respective repositories.

Although TAGs are designed to focus on specific areas, you can begin contributing through working groups (WGs) that operate under the TAGs with even more narrow scopes. WGs are community-driven groups that encourage ongoing collaboration in specific areas of interest.

> More details about the CNCF TAGs is available [here](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md). You can find the list of TAGs [here](https://github.com/cncf/toc/tree/main/tags).

Reviewer: Just a FYI, it really helps people using screen readers to not name the link text "here". Instead name the link text something descriptive that explains what they will find if they follow the link.

```suggestion
Learn more about [CNCF TAGs](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) and view a [list of the CNCF TAGs](https://github.com/cncf/toc/tree/main/tags).
```

Are you passionate about Cloud Native technologies and interested in starting a community group in your area? Look no further than the CNCF Meetups program! Check out the [CNCF Community Groups repository](https://github.com/cncf/communitygroups) for more information about the program, including best practices for running successful CNCF Community Groups.

### Ambassadors

[Cloud Native Ambassadors](https://www.cncf.io/people/ambassadors/) (CNAs) are individuals who are passionate about [Cloud Native Computing Foundation](https://www.cncf.io/) technology and projects, recognized for their expertise, and willing to help others learn about the framework and community.

Successful ambassadors are people such as bloggers, influencers, evangelists who are already engaged with a CNCF project in some way including contributing to forums, online groups, community events, etc.

> Details on the Ambassadors program, and information on how to join CNCF as an Ambassador is available [here](https://github.com/cncf/ambassadors).

Reviewer: Same comment as above, it would be more accessible to not use "here" in the link text.

Author: Sounds good. I've replaced it

## Projects

> The Cloud Native Computing Foundation projects are listed below, together with the brief information on contributing to them.

Reviewer: ```suggestion
The Cloud Native Computing Foundation projects are listed below with information on how to contribute.
```

- [Google Season of Docs](https://github.com/cncf/mentoring/tree/master/seasonofdocs) (GSoD)
- [Outreachy](https://github.com/cncf/mentoring/tree/master/outreachy)

If you are interested in participating in one of the programs, please check out the [CNCF mentoring repository](https://github.com/cncf/mentoring) for more details.

### Project Opportunities

The CNCF projects are always looking for new contributors. If you're interested in contributing to a CNCF project, check out the following:

- [CNCF Landscape](https://landscape.cncf.io/) - a map of all CNCF projects, along with their maturity level and their category
- [CLOTributor](https://clotributor.dev/) - a tool that helps you find beginner-friendly tasks in CNCF projects
- [Projects section below](#projects) lists all CNCF projects, with their primary language and brief information on contributing to them

### TAGs and WGs

CNCF Technical Advisory Groups (TAGs) oversee and coordinate the interests pertaining to a logical area of needs of end users and/or projects. More details about the CNCF TAGs is available [here](https://github.com/cncf/toc/tree/main/tags).

> Working groups (WGs) are the community-driven groups with the goal of continuous collaboration in the specific areas. CNCF WGs are created and curated by the CNCF TOC and driven by the community members. CNCF TOC repo provides more [details](https://github.com/cncf/toc/tree/master/workinggroups#cncf-working-groups) on the purpose and goals of WGs, together with the [list of them](https://github.com/cncf/toc/blob/master/README.md#working-groups).

Reviewer 2: I think we should remove the bit about WGs. You can see from the [TOC readme](https://github.com/cncf/toc) that there is only 1 standalone working group at the TOC level that is active (serverless). It might be better to say that many of the TAGs have WGs that people can get involved in, which puts the focus on finding a TAG to contribute to and then engaging with WGs within that TAG.

Author: Thanks for the feedback. I agree.

I've updated this section now. Can you have a new look?

| ------------------------------------------------------------ | ------------------------- | ------------------ |
|    [Cloud Native Glossary](projects/#cloud-native-glossary)        |           Definitions       |        Markdown      |     

## Archived Projects

> {{< projects level="archived" >}}

Reviewer: It's not clear to me why archived projects were removed? Was this a mistake?

Author: I've added a comment in this ticket, but it is lost.

I thought listing archived projects in a page where we encourage people to contribute to the projects is not good.
They cannot contribute to those projects anyway.

What do you think?

Reviewer: Are they listed somewhere else? I am just not comfortable removing that content entirely from the website since originally the project list was owned/managed by another group of people.

Author: They're at least listed here: https://contribute.cncf.io/contributors/projects/#archived-projects

I didn't touch that part. Only removed the archived projects in the https://contribute.cncf.io/contributors/ page, which a person is directed to when they click "I want to be or am a Contributor" link on [the homepage](https://contribute.cncf.io/)

It is not a big deal to list them back though :) Just let me know what you think

Reviewer: No thanks that helped, I'd forgotten that we had a separate page of projects listed elsewhere
